src/app/data-model/Nullable.h

```diff
@@ -74,6 +74,10 @@ struct Nullable : protected Optional<T>
         return true;
     }
 
+    // We think nullable struct fields / attributes won't be fabric scoped. As fabric scoped structs will be used in List encoding
+    // and Decoding, and lists won't contain null objects.
```

Comment:
I would somewhat prefer we handled this in the IsFabricScoped impl, because it's non-obvious when looking at that code that it will end up being affected by this.  That said, that can be a followup, and if we keep this here:
```suggestion
    // The only fabric-scoped objects in the spec are events and structs inside lists, and neither one can be nullable.
```